src/Testcontainers.EventStore/EventStoreConfiguration.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Testcontainers.EventStore;
+
+/// <inheritdoc cref="ContainerConfiguration" />
+[PublicAPI]
+public sealed class EventStoreConfiguration : ContainerConfiguration
+{
+  /// <summary>
+  /// Initializes a new instance of the <see cref="EventStoreConfiguration" /> class.
+  /// </summary>
+  /// <param name="username">The EventStore database.</param>
+  /// <param name="password">The EventStore username.</param>
+  public EventStoreConfiguration(
+    string username = null,
+    string password = null)
+  {
```

Comment:
The `EventStoreBuilder` does not expose APIs to configure the `Username` and `Password` property. Do we need them?

Reply:
No, we don't need them, I remove them.